sdk/batch/Microsoft.Azure.Batch/src/Common/BatchErrorCodeStrings.cs

```diff
@@ -599,56 +599,129 @@ public static class BatchErrorCodeStrings
+        /// </summary>
         public const string AllocationTimedout = "AllocationTimedout";
-
+        
+        /// <summary>
+        /// Desired number of nodes could not be allocated
+        /// </summary>
         public const string AllocationFailed = "AllocationFailed";
-
+        
+        /// <summary>
+        /// The specified account has reached core quota
+        /// </summary>
         public const string AccountCoreQuotaReached = "AccountCoreQuotaReached";
-
+        
+        /// <summary>
+        /// The specified account has reached low priority core quota
+        /// </summary>
         public const string AccountLowPriorityCoreQuotaReached = "AccountLowPriorityCoreQuotaReached";
-
+        
+        /// <summary>
+        /// The marketplace resource plan is missing required publisher property
+        /// </summary>
         public const string MissingPublisherPropertyInMarketPlacePlan = "MissingPublisherPropertyInMarketPlacePlan";
-
+        
+        /// <summary>
+        /// Allocation failed because the allocation condition is too restrictive. Please remove some constraints and try again
+        /// </summary>
         public const string OverconstrainedAllocationRequestError = "OverconstrainedAllocationRequestError";
-
+        
+        /// <summary>
+        /// Commmunication enabled pool has reached the maximum VM count
+        /// </summary>
         public const string CommunicationEnabledPoolReachedMaxVMCount = "CommunicationEnabledPoolReachedMaxVMCount";
-
+        
+        /// <summary>
+        /// The specified account has reached spot core quota
+        /// </summary>
         public const string AccountSpotCoreQuotaReached = "AccountSpotCoreQuotaReached";
-
+        
+        /// <summary>
+        /// Pool creation failed with error 'Failed to perform resource identity operation'
+        /// </summary>
         public const string FailedIdentityOperation = "FailedIdentityOperation";
-
+        
+        /// <summary>
+        /// Pool creation failed with disk encryption error
+        /// </summary>
         public const string DiskEncryptionFailure = "DiskEncryptionFailure";
-
+        
+        /// <summary>
+        /// A dependent resource specified for the Pool was not found.
+        /// </summary>
         public const string DependentResourceNotFound = "DependentResourceNotFound";
-
+        
+        /// <summary>
+        /// Assigned policy on resource has blocked pool operation.
+        /// </summary>
         public const string OperationsRestrictedByPolicy = "OperationsRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on PublicIP has blocked pool operation.
+        /// </summary>
         public const string OperationsOnPublicIPRestrictedByPolicy = "OperationsOnPublicIPRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on Custom Image has blocked pool operation.
```

Comment:
Would this ever happen for marketplace images

Reply:
This will only happen for BYOImage/Custom Image (including managed image or SIG image) cases, so this will not happen for marketplace imaages